core/server/middleware/labs.js

```diff
@@ -7,7 +7,7 @@ labs = {
         if (labsUtil.isSet('subscribers') === true) {
             return next();
         } else {
-            return errors.handleAPIError(new errors.NotFoundError(), req, res, next);
+            return next(new errors.NotFoundError());
```

Comment:
i hope that helps:
https://github.com/TryGhost/Ghost/pull/7431/commits/da4ce0a9e37339c4494b33266ea23b8142a8ec47